<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="Lands_Info">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>860</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="Lands_archieve">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>850</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_119">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>المدينه</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_38">
     <property name="geometry">
      <rect>
       <x>786</x>
       <y>20</y>
       <width>150</width>
       <height>39</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_117">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>البلوك</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_18">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>69</y>
       <width>1030</width>
       <height>760</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>المدينه</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>الحى </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>البلوك</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>رقم القطعه </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>نوع الارض</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>السعر</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>المساحه</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>حالة العقار</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>باقي الاقساط</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>الربح </string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btn_search_14">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>بحث</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_118">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>الحى</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_37">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>150</width>
       <height>39</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_39">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>141</width>
       <height>39</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>طباعه</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Lands">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>860</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_309">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>720</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>حالة العقار</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add_land">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>780</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>إضافة</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_remove_land">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>780</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>حذف</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chxother">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>470</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>اماكن اخري</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_edit_land">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>780</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>تعديل</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtdetails">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>441</width>
       <height>201</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="placeholderText">
      <string> تفاصيل اخرى عن الارض</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_172">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>340</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>إسم المالك</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_171">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>410</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>رقم الهاتف الاساسي</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtprice">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>590</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtarea">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>650</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtoffer">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="chxmahgoz">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>470</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>أرض محجوز</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_263">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>262</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>البلوك</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_174">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>530</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>نوع الارض</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtpremium">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>770</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_259">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>660</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>المساحة (متر مربع)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_271">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>190</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>الحي</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_175">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>470</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>حالة الارض</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtphone">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>400</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxtypeland">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>520</y>
       <width>330</width>
       <height>49</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>اكثر تميزا</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>اجتماعي</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>صناعي</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>بيت وطن</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>سكني</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>خدمي</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="chxover">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>الاوفر</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_258">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>590</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>السعر</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_173">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>رقم القطعة</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxstatus">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>710</y>
       <width>331</width>
       <height>49</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>خالص الثمن </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>باقي قسط</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="txtpiece">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>60</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_272">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>780</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>باقي الاقساط</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxStreet">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>190</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxBlock">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>260</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxCity">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>130</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_273">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>130</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>المدينة</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>441</width>
       <height>451</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="title">
      <string>صور المصنع</string>
     </property>
     <widget class="QPushButton" name="btn_add_img">
      <property name="geometry">
       <rect>
        <x>376</x>
        <y>400</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>simplified arabic</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_remove_img">
      <property name="geometry">
       <rect>
        <x>336</x>
        <y>400</y>
        <width>35</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>simplified arabic</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblImgs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>441</width>
        <height>421</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>الصورة</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btn_show_imgs">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>400</y>
        <width>61</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>simplified arabic</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>عرض</string>
      </property>
     </widget>
     <zorder>tblImgs</zorder>
     <zorder>btn_add_img</zorder>
     <zorder>btn_remove_img</zorder>
     <zorder>btn_show_imgs</zorder>
    </widget>
    <widget class="QLineEdit" name="lineEdit_58">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>330</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="lands_list">
    <property name="geometry">
     <rect>
      <x>1031</x>
      <y>0</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 24px;</string>
    </property>
    <item>
     <property name="text">
      <string>  الأراضى</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>  أرشيف الأراضى</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
